src/admin/resolvers/queries/ApprovedItem.ts

```diff
@@ -76,3 +77,53 @@ export async function getApprovedItemByUrl(
 
   return await dbGetApprovedItemByUrl(context.db, args.url);
 }
+
+/**
+ * Retrieves a history of when and on what scheduled surface a corpus item
+ * was scheduled.
+ */
+export async function getScheduledSurfaceHistory(
+  parent,
+  args,
+  { db }
+): Promise<ScheduledItem[]> {
+  // Get the external ID of the Approved Item itself
+  const { externalId } = parent;
+
+  // Get the optional filters. Not specifying any means we will retrieve
+  // the entire list of all the occasions this approved item was scheduled
+  // on all surfaces, in descending order (most recently scheduled first).
+  //
+  // Limiting it to a single surface means we'll only retrieve data for
+  // one scheduled surface (useful for when you only need the one to filter out
+  // prospects on the Prospecting page).
+  //
+  // Limiting the number of results to one means only the most recent result
+  // will be returned.
+  const { scheduledSurfaceGuid, limit } = args;
```

Comment:
It seems unlikely that there would be a ton of entries to fetch, but I would recommend putting a default limit in place.